CHANGELOG-hmp-156-update-text-for-access-permissions-for-workspaces.md

```diff
@@ -0,0 +1,3 @@
+- Render "The workspaces feature is only available if logged in and is part of the allowed Globus group." for workspaces page if user is not logged in.
+- Render "You must be a member of the allowed Globus group to access this feature. " if user is logged in but not a member of the allowed Globus group.
+- Render Workspace page if user is logged in and is a member of the allowed Globus group.
```

Comment:
I think the changelog entries are a little too granular and could potentially be confusing to someone reading through them.

Perhaps

> - Update workspaces landing page messaging to instruct users without membership in the workspaces Globus group to request access.